bundles/action/org.openhab.action.astro/META-INF/MANIFEST.MF

```diff
@@ -11,6 +11,8 @@ Bundle-ManifestVersion: 2
 Bundle-Description: This is the Astro action of the open Home Aut
  omation Bus (openHAB)
 Import-Package: org.apache.commons.lang.builder,
+ org.openhab.binding.astro.internal.calc,
```

Comment:
Absolutely right and we should change this (out of the scope of this Bugfix). What i changed was simply removing the require-bundle deep and add the import-package instead. @theoweiss could you please take care of this and create another PR moving the externally required classes in the appropriate package?